openapi/refs.go

```diff
@@ -7,11 +7,11 @@ import (
 )
 
 // readRef takes a reference and attempts to unmarshal it's content into the struct being passed as `i`.
-func readRef(filename string, i interface{}) error {
-	fmt.Printf("Reading reference: %s\n", filename)
-	content, err := os.ReadFile(filename)
+func readRef(absFilename string, i interface{}) error {
```

Comment:
Cool. qq: You mentioned that we should move to absolute paths for refs. Do you intend to pass that on to our users? ie. Are you saying users of can should ensure all refs in their documentation are pathed absolutely?

Reply:
no. I feel that all paths internally in the application should be absolute for simplicity's sake